providers/ethereum-provider/src/EthereumProvider.ts

```diff
@@ -18,13 +18,17 @@ export type QrModalOptions = Pick<
   | "chainImages"
   | "desktopWallets"
   | "enableExplorer"
-  | "explorerAllowList"
-  | "explorerDenyList"
+  | "explorerRecommendedWalletIds"
+  | "explorerExcludedWalletIds"
   | "mobileWallets"
   | "privacyPolicyUrl"
   | "termsOfServiceUrl"
   | "tokenImages"
+  | "tokenContracts"
```

Comment:
`tokenContracts`, `standaloneChains`, `enableNetworkView`, `enableAccountView` options are not available here (standalone modal).

Reply:
f0e702a55c4c21f2d7bd32df61ef10c044bb66ee 


oh i didn't know thank you for telling me. Corrected this. Can you check when you have time?